modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -171,7 +171,7 @@ struct IEUnit {
     static const char *name() { return "IEModelConfig"; }
 
     cv::gapi::ie::detail::ParamDesc params;
-    IE::CNNNetwork net;
+    mutable IE::CNNNetwork net;
```

Comment:
Is it possible to make a graph non-const [here](https://github.com/opencv/opencv/pull/18240/files#diff-c3ec2ae59c90eaf62d3547ff931a31a3R443) to avoid adding `mutable`?

Reply:
Deprecated.